draft-ietf-rtcweb-jsep.xml

```diff
@@ -2621,19 +2621,20 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     only included if such an option was present in the
                     offer.</t>
 
-                    <t>The next step is to generate lip sync groups as
-                    defined in <xref target="RFC5888" />, Section 7. For
-                    each MediaStream with more than one referenced
-                    RtpTransceiver, a group of type "LS" MUST be added
-                    that contains the mid values for each RtpTransceiver
-                    added with that MediaStream. In some cases this may
-                    result in adding a mid to a given LS group that was
-                    not in that LS group in the associated
-                    offer. Although this is not allowed by <xref
-                    target="RFC5888" />, it is allowed when implementing
-                    this specification. [[OPEN ISSUE: This is still
-                    under discussion. See:
-                    https://github.com/rtcweb-wg/jsep/issues/162.]]</t>
+                    <t>The next step is to generate session-level lip
+                    sync groups as defined in <xref target="RFC5888"/>,
+                    Section 7. For each group of type "LS" present in the offer,
+                    determine which of the local RtpTransceivers identified by
+                    the group's mid values reference a common local MediaStream
```

Comment:
changed to 'that'; agree that's slightly more specific
which is plural here, I believe